psh/tools/common.py

```diff
@@ -38,6 +38,25 @@ def create_testdir(p, dirname):
     psh.assert_cmd(p, f'mkdir {dirname}', '', msg)
 
 
+def assert_timestamp_change(p, touched_files, uptimes, dir=''):
```

Comment:
if this is a **common** routine, remove all references to the `touch` test (eg. variable names)